src/App.tsx

```diff
@@ -87,6 +88,16 @@ const App: React.FC = () => {
                 </ProtectedRoute>
               }
             />
+            <Route
+              path="/pairing"
+              element={
+                <ProtectedRoute>
+                  <DevRoute>
```

Comment:
Put this behind admin ? We can merge my pr then put admin instead of dev.

Reply:
You marked this as resolved but it' still devroute, do you want to leave it as devroute?

Reply:
no that was accident haha